test/train-sets/ref/automl_readable.txt

```diff
@@ -1,5 +1,5 @@
 Version 8.11.0
```

Comment:
Will this have the right version once VW 9 is released?

Reply:
When version.txt changes then these files are updated